src/endpoint/s3/s3_rest.js

```diff
@@ -96,6 +96,12 @@ async function handle_request(req, res) {
         error_token_expired: S3Error.ExpiredToken,
         auth_token: () => signature_utils.make_auth_token_from_request(req)
     };
+    // AWS s3 returns an empty response when s3 request sends without host header.
+    if (!req.headers.host) {
+        dbg.warn('s3_rest: handle_request: S3 request is missing host header, header ', req.headers);
```

Comment:
Suggesting to clarify that those are the current headers that are printed.
```suggestion
        dbg.warn('s3_rest: handle_request: S3 request is missing host header, request headers ', req.headers);
```

Reply:
I think only adding header also should be fine

Reply:
I just think there is a typo there "header" instead of "headers".